<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>348</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QTabWidget" name="editorTabs">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>581</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>5</verstretch>
     </sizepolicy>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>235</y>
      <width>473</width>
      <height>229</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>2</verstretch>
     </sizepolicy>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>الرسائل</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextBrowser" name="outputView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>451</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>المتغيرات</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>ملف</string>
    </property>
    <widget class="QMenu" name="mnuRecentFiles_2">
     <property name="title">
      <string>ملفات سابقة</string>
     </property>
     <addaction name="actionDummy"/>
    </widget>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="mnuRecentFiles_2"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="mnuEdit">
    <property name="title">
     <string>تحرير</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_edit_cut"/>
    <addaction name="action_edit_copy"/>
    <addaction name="action_edit_paste"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
    <addaction name="action_replace"/>
   </widget>
   <widget class="QMenu" name="mnuPRogram">
    <property name="title">
     <string>برنامج</string>
    </property>
    <addaction name="mnuProgramRun"/>
    <addaction name="action_verify"/>
    <addaction name="garbageCollect"/>
    <addaction name="action_wonderfulmonitor"/>
    <addaction name="actionSpeedFast"/>
    <addaction name="actionSpeedMedium"/>
    <addaction name="actionSpeedSlow"/>
    <addaction name="actionSpeedSuper"/>
   </widget>
   <widget class="QMenu" name="mnuHelp">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>مساعدة</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionKalimatManual"/>
    <addaction name="action_help_keyboard"/>
    <addaction name="separator"/>
    <addaction name="action_walkthroughs"/>
    <addaction name="action_report_bug"/>
    <addaction name="action_contact_us"/>
    <addaction name="separator"/>
    <addaction name="action_about_kalimat"/>
   </widget>
   <widget class="QMenu" name="mnuTest">
    <property name="title">
     <string>اختبار</string>
    </property>
    <addaction name="actionLexize"/>
    <addaction name="actionParse"/>
    <addaction name="actionCompile"/>
    <addaction name="actionCompile_without_tags"/>
    <addaction name="actionEvaluate"/>
    <addaction name="actionGo_to_position"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>رمز خاص</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action"/>
    <addaction name="action_4"/>
    <addaction name="action_3"/>
    <addaction name="action_6"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuEdit"/>
   <addaction name="mnuPRogram"/>
   <addaction name="mnuTest"/>
   <addaction name="menu"/>
   <addaction name="mnuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="mnuProgramRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockSearchReplace">
   <property name="minimumSize">
    <size>
     <width>79</width>
     <height>100</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ابحث عن</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>واستبدل به</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSearchString">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>10</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtReplacementString">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>40</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFindPrev">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFindNext">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReplacePrev">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReplaceNext">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFindStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionLexize">
   <property name="text">
    <string>Lexize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionParse">
   <property name="text">
    <string>Parse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionEvaluate">
   <property name="text">
    <string>Evaluate</string>
   </property>
  </action>
  <action name="actionKalimatManual">
   <property name="text">
    <string>البرمجة بكلمات</string>
   </property>
  </action>
  <action name="action_walkthroughs">
   <property name="text">
    <string>محاضرات عملية</string>
   </property>
  </action>
  <action name="action_report_bug">
   <property name="text">
    <string>إبلاغ عن خطأ بالبرنامج</string>
   </property>
  </action>
  <action name="action_contact_us">
   <property name="text">
    <string>اتصل بنا</string>
   </property>
  </action>
  <action name="action_about_kalimat">
   <property name="text">
    <string>عن كلمات...</string>
   </property>
  </action>
  <action name="action_help_keyboard">
   <property name="text">
    <string>العلامات الخاصة ولوحة المفاتيح</string>
   </property>
  </action>
  <action name="mnuProgramRun">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/runIcon.png</normaloff>:/icons/icons/runIcon.png</iconset>
   </property>
   <property name="text">
    <string>تنفيذ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/saveIcon.png</normaloff>:/icons/icons/saveIcon.png</iconset>
   </property>
   <property name="text">
    <string>حفظ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/openIcon.png</normaloff>:/icons/icons/openIcon.png</iconset>
   </property>
   <property name="text">
    <string>فتح</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_verify">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/checkIcon.png</normaloff>:/icons/icons/checkIcon.png</iconset>
   </property>
   <property name="text">
    <string>تحقّق</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/newIcon.png</normaloff>:/icons/icons/newIcon.png</iconset>
   </property>
   <property name="text">
    <string>جديد</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="text">
    <string>حفظ بإسم...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>خروج</string>
   </property>
  </action>
  <action name="action_edit_cut">
   <property name="text">
    <string>قص</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_edit_copy">
   <property name="text">
    <string>نسخ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_edit_paste">
   <property name="text">
    <string>لصق</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>مسح</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>تراجع</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>إعادة</string>
   </property>
  </action>
  <action name="garbageCollect">
   <property name="text">
    <string>جمع المهملات</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="text">
    <string>dummy</string>
   </property>
  </action>
  <action name="action_wonderfulmonitor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>المراقب العجيب</string>
   </property>
  </action>
  <action name="action_shift">
   <property name="text">
    <string>&lt;     (shift+ج)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>&gt;     (shift+د)</string>
   </property>
  </action>
  <action name="action_shift_2">
   <property name="text">
    <string>،     (shift+ن)</string>
   </property>
  </action>
  <action name="action_shift_3">
   <property name="text">
    <string>[     (shift+ب)</string>
   </property>
  </action>
  <action name="action_shift_4">
   <property name="text">
    <string>]     (shift+ي)</string>
   </property>
  </action>
  <action name="action_shift_5">
   <property name="text">
    <string>.     (shift+ز)</string>
   </property>
  </action>
  <action name="action_shift_6">
   <property name="text">
    <string>إ     (shift+غ)</string>
   </property>
  </action>
  <action name="action_find">
   <property name="text">
    <string>بحث...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_replace">
   <property name="text">
    <string>استبدال...</string>
   </property>
  </action>
  <action name="actionSpeedFast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>سريع</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionSpeedMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>متوسط</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionSpeedSlow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>بطيء</string>
   </property>
   <property name="shortcut">
    <string>Alt+Z</string>
   </property>
  </action>
  <action name="actionCompile_without_tags">
   <property name="text">
    <string>Compile without tags</string>
   </property>
  </action>
  <action name="actionGo_to_position">
   <property name="text">
    <string>Go to position</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSpeedSuper">
   <property name="text">
    <string>خارق</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>,</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>[</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>{</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>}</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>mainWindowClosed(QCloseEvent*)</signal>
 </slots>
</ui>
